src/cli.js

```diff
@@ -8,6 +10,10 @@ const run = () => {
 	const argv = yargs
 		.commandDir('commands') // commands are in the './commands/' dir
 		.demandCommand() // require a command to be used - no default behavior
+		.option('config', {
```

Comment:
alternatively, you could build the `package.config` default setting into [buildCommands/config/env.js](https://github.com/meetup/mwp-cli/blob/master/src/commands/buildCommands/config/env.js), which is the file that reads app-specific configuration from `config.{env}.json` - seems reasonable to augment that with data from `package.json`, and then it can be formally validated and kept tightly coupled with related config settings.